scripts/index.js

```diff
@@ -0,0 +1,185 @@
+        nodesSecond.forEach(node => replaceClass(node, firstClass, secondClass));
+    };
+
+    const changeCase = () => {
+        const lowCase = 'down';
+        const upCase = 'up';
+        switchClasses(lowCase, upCase);
+    };
+
+    const saveLang = (lang) => {
+        localStorage.setItem('lang', lang);
+    };
+
+    const changeLang = () => {
+        const onLang = 'on';
+        const offLang = 'off';
+        switchClasses(onLang, offLang);
+        saveLang(detectLang());
+    };
+
+    const setLang = () => {
+        const isLocal = localStorage.getItem('lang');
+        if (isLocal) {
+            if (!(isLocal === detectLang())) {
+                changeLang();
+            }
+        }
+    };
+
+    setLang();
+
+    const changeActiveClass = (button) => {
+        button.classList.contains(animationClass) ? button.classList.remove(animationClass) : button.classList.add(animationClass);
+    };
+
+    const pressButton = (className) => {
+        let button;
+        if (document.querySelector(`.${className}`)) {
+            button = document.querySelector(`.${className}`).parentNode;
+        } else {
+            return;
+        }
+        changeActiveClass(button);
+    };
+
+    const isRunOnKeys = (eo) => {
+        pressed.add(eo.code);
+
+        for (let code of codes) {
+            if (!pressed.has(code)) {
+                return;
+            }
+        }
+        pressed.clear();
+        changeLang();
+        return true;
+    };
+
+    const hotKey = {
+        ShiftLeft: null,
+        AltLeft: null,
+        AltRight: null,
+        ShiftRight: null,
+    };
+
+    const keyEventUp = (eo) => {
+        textArea.focus();
+        pressButton(eo.code);
+        if (eo.code in hotKey) {
+            if (isRunOnKeys(eo)) {
+                eo.preventDefault();
+                return;
+            }
+        }
+        if (funcButton[eo.code]) {
+            funcButton[eo.code]();
+            eo.preventDefault();
+        }
+    };
+
+    const keyEventDown = (eo) => {
+        pressButton(eo.code);
+        if (eo.code in hotKey) {
+            pressed.delete(eo.code);
+            eo.preventDefault();
+        }
+        if (funcButton[eo.code]) {
+            eo.preventDefault();
+        }
+    };
+
+    const clickAnim = (elem) => {
+        changeActiveClass(elem);
+        setTimeout(() => changeActiveClass(elem), 100);
+    };
+
+    const clickButton = (eo) => {
+        textArea.focus();
+        let elem = eo.target;
+        if (elem.classList.contains('down')) {
+            elem = elem.parentNode.parentNode;
+        }
+        clickAnim(elem);
+        const sign = elem.querySelector('.down').innerHTML;
+        if (!exceptSymbols[sign]) {
+            textArea.value += sign;
+            return;
+        }
+        if(funcButton[sign]) {
+            funcButton[sign]();
+        }
+    };
+
+    keys.forEach(elem => {
```

Comment:
You can read about event delegation in order to not add event listener to each key